### ISSUE TITLE
httpd didnt work

### ISSUE THREAD

**Issue by mrharmonies** [rmyorston/busybox-w32#226]
httpd didnt work on my system (windows 10). Nothing happen when i write httpd -p 80 -h . -v

**Comment by rmyorston** [comment]
It's working for me. I used a recently installed 64-bit Windows 10 system. From one instance of busybox-32 I ran the server in the foreground (```-f```) so we can see the log messages:
```
~ $ cat index.html
<html><body>Hello world!</body></html>
~ $ httpd -p 80 -h . -vv -f
```
Since this was the first time I'd run the web server on this virtual machine the Windows firewall asked if busybox-w32 should be allowed network access: I said OK.

From another busybox-w32 instance I connected to the server:
```
~ $ wget -O - http://localhost/
Connecting to localhost (127.0.0.1:80)
writing to stdout
<html><body>Hello world!</body></html>
written to stdout
~ $
```
The server said:
```
[::ffff:127.0.0.1]:50319: url:/
[::ffff:127.0.0.1]:50319: response:200
```
I was also able to access the server from a different machine using a web browser. It all seems to work as intended.
